src/pages/dongwheeKim/Login/Login.js

```diff
@@ -2,8 +2,105 @@ import React from 'react';
 import './Login.scss';
 
 class LoginDongWhee extends React.Component {
+  goToMain = () => {
+    fetch('http://10.58.5.59:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.textId,
+        password: this.state.textPw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => console.log('결과: ', result));
+    this.props.history.push('/main-whee');
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      textId: '',
+      textPw: '',
+      loginButtonColor: false,
+    };
+  }
+
+  handleIdInput = event => {
+    this.setState({
+      textId: event.target.value,
+    });
+  };
+
+  handlePwInput = event => {
+    this.setState({
+      textPw: event.target.value,
+    });
+  };
```

Comment:
계산된 속성명을 활용해 다음과 같이 작성할 수 있습니다!
대신, state의 각 key값에 맞는 name attribute를 각각의 input 태그에 부여해주셔야 합니다.

```suggestion
  handleInput = event => {
    this.setState({
      [event.target.name] : event.target.value,
    });
  };
```

Reply:
넵 수정했습니다!